Dockerfile

```diff
@@ -57,11 +57,18 @@ RUN mkdir /src && \
     luarocks install lua-requests
 
 ADD . /src/
+
+WORKDIR /src/samples
+RUN make
```

Comment:
It would be useful to pass an environment variable to make such as 
```
make SRC=/src/samples/1_example
```
Which would build those files and link them as `train.lbjc`, `init.lbjc` and `output.lbjc` when creating the container

Reply:
Currently all samples in /src/samples will be compiled producing train.lbjc, init.lbjc and output.lbjc in the folders lib under each sample e.g. /src/samples/{sample_X}/lib
this approach allows to activate any sample without need to rebuild the container. To activate the sample you just pass environment variable activate_app: 
`docker run -e activate_app="1_getstarted" -p 3000:3000 -it webtorch`